components/HeroSection.vue

```diff
@@ -18,19 +18,19 @@
         >
           <v-heading
             class="hero-section-heading"
-            color="secondary"
             line-height="1.2"
-            :weight="400"
+            :weight="500"
           >
-            Welcome to the Arcana Alpha Testnet
+            Welcome to the
           </v-heading>
           <v-heading
             class="hero-section-heading-em"
             :level="1"
             :weight="700"
             line-height="1.2"
           >
-            Quantum Descent
+            Arcana Beta Testnet<br />
```

Comment:
Can we avoid the hard line break? If needed, you can use a non-blocking space -- `&nbsp;` -- between the words.